src/amber/router/context.cr

```diff
@@ -11,9 +11,11 @@ class HTTP::Server::Context
 
   property route : Radix::Result(Amber::Route)
   getter router : Amber::Router::Router
+  getter cookies : Amber::Router::Cookies::Store
 
   def initialize(@request : HTTP::Request, @response : HTTP::Server::Response)
     @router = Amber::Router::Router.instance
+    @cookies = Amber::Router::Cookies::Store.build(request, Amber::Server.key_generator)
```

Comment:
Cool. That's what I shared above was from rails middleware. Lets just make it nilable here and controller base and create a cookie pipe in another PR where we refactor session/flash to write to the same cookie like rails. This should speed up those pipes as well.
```cr
getter cookies : Amber::Router::Cookies::Store?
```